Rocket.Chat/de.lproj/Localizable.strings

```diff
@@ -132,3 +132,12 @@
 "chat.messages.list.empty" = "Keine Nachrichten gefunden";
 "chat.messages.pinned.list.title" = "Angeheftete Nachrichten";
 "chat.messages.starred.list.title" = "Favorisierte Nachrichten";
+
+// New Room List
+"new_room.cell.public_channel.title" = "Öffentlicher Kanal"
+"new_room.cell.public_chanell.description" = "Jeder Benutzer kann einem öffentlichen Raum beitreten.";
+"new_room.cell.read_only.title" = "Schreibgeschützter Kanal"
+"new_room.cell.read_only.description" = "Nur Moderatoren oder Admins können in schreibgeschützten Kanälen schreiben";
+"new_room.group.channel.name" = "Channel Name"
```

Comment:
Some missing semi-colons preventing build

Reply:
Oh man, sry for this. I correct it ASAP.